index.html

```diff
@@ -0,0 +1,111 @@
+          </p>
+          <p>EDX en colaboración con Harvard</p>
+          <p><b>Estudios de danza clásica y latinoamericana</b></p>
+          <p>
+            Escuela de ballet de Chía Ariadna Páramo - Incolballet - Danzar Colombia
+            - Vasilliev Academy
+          </p>
+        </div>
+        <div>
+          <h3>Manejo de lenguajes de programación y programas</h3>
+          <p>Java - Java FX - Python - AutoCAD - Inventor - R-Studio</p>
+        </div>
+      </div>
+      <div>
+        <div>
+          <h3>Experiencia laboral</h3>
+          <p>
+            <b>Modelos de las Naciones Unidas (ONU)</b>
+          </p>
+          <p>Colegio Marymount - Gimnasio Campestre</p>
+          <p>
+            <b>Trabajo social Fundación MIA</b>
+          </p>
+          <p>
+            <b>Trabajo social Sabana</b>
+          </p>
+          <p>Macroproyecto de transformación académica Universidad de la Sabana</p>
+          <p><b>Programa de tutoría para becarios</b></p>
+          <p>Universidad de la Sabana</p>
+          <p>
+            <b> Propuesta de desarrollo tecnológico y de visibilidad </b>
+          </p>
+          <p>Parque Fundación Jaime Duque</p>
+          <p>
+            <b>
+              Proyecto ambiental de prevención de envenenamiento animal: Collar
+              inteligente
+            </b>
+          </p>
+          <p>
+            <b>
+              Proyecto de STEAM: implementación de un domo de estudio</b></p>
+              <p>
+                Colegio SEK Colombia
+            </b>
+          </p>
+          </div>
+      </div>
+        <div>
+          <h3>Habilidades</h3>
+          <ul>
+            <li>Planeación</li>
+            <li>Liderazgo</li>
+            <li>Gestión de proyectos</li>
+            <li>Adaptabilidad</li>
+            <li>Recursividad</li>
+            <li>Resolución de problemas</li>
+            <li>Pensamiento creativo crítico y estratégico</li>
+            <li>Atención a los detalles</li>
+            <li>Buena comunicación</li>
+            <li>Trabajo en equipo</li>
+          </ul>
+        </div>
+        <div>
+          <h3>Intereses y Hobbies</h3>
+          <p><b>Técnicos:</b> Investigación de la programación e inteligencia artificial aplicada a la robótica, aportes
+            de la tecnología al desarrollo científico (NASA) y a problemas de la sociedad (salud, medio ambiente, desigualdad/brecha tecnológica, seguridad),
+              desarrollo de superapps y páginas web desde la perspectiva del diseño para el cliente, modernización y facilitación de uso de las páginas web y bases
+              de datos de los diferentes servicios disponibles al público.</p>
+          <p><b>Artísticos:</b> Dibujo, cerámica, composición musical, interpretación dramática y artística, danzas de carácter, contemporánea, clásica y salsa</p>
+          <p><b>Idiomas:</b> Español, Inglés, Francés (nivel medio), Alemán (inicial)</p>

```

Comment:
Los intereses y hobbies se están enunciando directamente como un listado, así que lo más adecuado es ponerlos dentro de una lista.

El efecto de negrilla lo puedes lograr encerrando el texto que deseas dentro de una etiqueta <span> y a esa etiqueta le asignas una clase para posteriormente mostrar los textos en negrilla
```suggestion
          <p><span class="my-class-for-bold">Idiomas:</span> Español, Inglés, Francés (nivel medio), Alemán (inicial)</p>
          
          ...
          
          .bold {
              font-weight: bold;
          }

```